source/frontend/src/hooks/repositories/useMapProperties.tsx

```diff
@@ -20,6 +20,7 @@ export const useMapProperties = () => {
         STREET_ADDRESS_1: params?.STREET_ADDRESS_1,
         PID_PADDED: params?.PID?.replace(/[-\s]/g, ''),
         PIN: params?.PIN,
+        PROPERTY_ID: params?.PROPERTY_ID,
```

Comment:
nit: PIMS_PROPERTY_ID would make it explicit that we are loading something from our side.